recipes/full_finetune_distributed.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, cfg: DictConfig) -> None:
 
         if (
             cfg.get("fsdp_cpu_offload", False)
-            and cfg.get("fused", False)
+            and cfg.optimizer.get("fused", False)
```

Comment:
It's fine to directly check the optimizer here, b/c we require it for any configs trying to use this recipe.

Reply:
Technically shouldn't we check that device='cuda'? Though it's probably unlikely that you'd train with anything else.